virt/virtual_machines/importing_vms/virt-importing-virtual-machine-images-datavolumes.adoc

```diff
@@ -4,19 +4,15 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-importing-virtual-machine-images-datavolumes
 toc::[]
 
-You can import an existing virtual machine image into your {product-title}
-cluster. {VirtProductName} uses DataVolumes to automate the import of data and the
-creation of an underlying PersistentVolumeClaim (PVC).
+Use the Containerized Data Importer (CDI) to import a virtual machine image into a PersistentVolumeClaim (PVC) by using a DataVolume. You can attach a DataVolume to a virtual machine for persistent storage.
```

Comment:
s/PersistentVolumeClaim/persistent volume claim 

(or backticks as-is + "object")